<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugsDb::Internal::DrugsDbModeWidget</class>
 <widget class="QWidget" name="DrugsDb::Internal::DrugsDbModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="2" column="0" rowspan="3" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="availableTreeWidget">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="download">
         <property name="text">
          <string>Download sources</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="createDatapack">
         <property name="text">
          <string>Create DataPack</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="downloadSPC">
         <property name="text">
          <string>Download SPC</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="seeComponents">
         <property name="toolTip">
          <string>To see components, you must run at least one time the database creation and populate it with the drugs.</string>
         </property>
         <property name="text">
          <string>Show components</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Output database information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Filename</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="sizeLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="fileNameLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="dateLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Path</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="path">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="nonFreeGroup">
         <property name="title">
          <string>Non-free part</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="addAtc">
            <property name="text">
             <string>Add ATC to database</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="linkMols">
            <property name="text">
             <string>Link molecules with ATC</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="addDDI">
            <property name="text">
             <string>Add drug-drug interactions</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="addPims">
            <property name="text">
             <string>Add PIMs</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="addPreg">
            <property name="text">
             <string>Add pregnancy checking</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="spc">
            <property name="text">
             <string>Download SPC</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="seeDbDescription">
         <property name="text">
          <string>See database description file</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="seeDatapackDescription">
         <property name="text">
          <string>See datapack description file</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="freeGroup">
         <property name="title">
          <string>Free part</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="unzip">
            <property name="text">
             <string>Unzip sources</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="prepare">
            <property name="text">
             <string>Prepare raw sources</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="createDb">
            <property name="text">
             <string>Create drugs database</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="populate">
            <property name="text">
             <string>Populate database with drugs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="showDbReport">
         <property name="text">
          <string>Show database report</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Utils::PathChooser" name="sourceSelector" native="true"/>
       </item>
      </layout>
      <zorder>sourceSelector</zorder>
      <zorder>downloadSPC</zorder>
      <zorder>showDbReport</zorder>
      <zorder>seeComponents</zorder>
      <zorder>download</zorder>
      <zorder>createDatapack</zorder>
      <zorder>freeGroup</zorder>
      <zorder>nonFreeGroup</zorder>
      <zorder>progressBar</zorder>
      <zorder>groupBox_3</zorder>
      <zorder>seeDbDescription</zorder>
      <zorder>seeDatapackDescription</zorder>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>535</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="startJobs">
     <property name="text">
      <string>Start jobs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/widgets/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
